bob-backend/src/core.rs

```diff
@@ -257,16 +259,16 @@ impl Backend {
                 self.put_single(key, data, op).await?;
             }
             Ok(())
-        } else if let Some(path) = disk_path {
+        } else if let Some(paths) = disk_paths {
             debug!(
                 "remote nodes is empty, /{:.3}ms/",
                 sw.elapsed().as_secs_f64() * 1000.0
             );
-            let res = self
-                .put_single(key, data, Operation::new_local(vdisk_id, path))
-                .await;
-            trace!("put single, /{:.3}ms/", sw.elapsed().as_secs_f64() * 1000.0);
-            res
+            for p in paths {
+                self.put_single(key, data, Operation::new_local(vdisk_id, p)).await?;
+                trace!("put single, /{:.3}ms/", sw.elapsed().as_secs_f64() * 1000.0);
```

Comment:
This log is for the entire operation, not per drive, so you should move out it from the loop